general/development/policies/backporting.md

```diff
@@ -49,20 +51,19 @@ Many issues can be appropriately classified as borderline bug-fix/improvements.
 ## Backport fixes to unsupported branches
 
 - Given the [general policy](#general-policy) above, only supported stable branches are candidates normally.
-- Also security, privacy, data-loss and regressions caused by any of the previous issue types are accepted to be fixed into security-only supported branches.
-
-:::note
+- In addition to this any security, privacy, data-loss, and regression, caused by any of the previous issue types are accepted to be fixed into security-only supported branches.
```

Comment:
Is the comma after "data-loss" needed/correct? And, at the same time... shouldn't a comma be added after "previous issue types"?

Note this is how I'd say it in Spanish, heh. Not sure if the same rules apply. :-)

Other than that, my +1 to this. I think all the points in the policy remains correct.

Reply:
It's a separator in a list whose items are:
- security
- privacy
- data-loss
- regression

It's known as an Oxford comma. Personally I always use it, but others do not. It's neither right, nor wrong.